scripts/create_qdrant_collection.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+# Configuration
+QDRANT_HOST="qdrant"
+QDRANT_PORT="6333"
+COLLECTION_NAME="default"
```

Comment:
add as parameters, see script call

- advantage: can be used outside docker compose